scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -1140,7 +1140,8 @@ object TreeSyntax {
     // Multiples and optionals
     implicit def syntaxArgs: Syntax[List[Term]] = Syntax {
       case (b: Term.Block) :: Nil => s(" ", b)
-      case (f @ Term.Function(params, _)) :: Nil if !params.exists(_.decltpe.isEmpty) =>
```

Comment:
forgot to ask: why this change?

Reply:
It was adding {} for `apply( (a: String) => a )` which is not neccessary and was breaking one of the added tests, since {} would be a different tree